Lib/threading.py

```diff
@@ -369,15 +369,13 @@ def notify(self, n=1):
         if not self._is_owned():
             raise RuntimeError("cannot notify on un-acquired lock")
         all_waiters = self._waiters
-        waiters_to_notify = _deque(_islice(all_waiters, n))
```

Comment:
After removing `waiters_to_notify`, `all_waiters` looks bit ugly.
Please rename `all_waiters` to just `waiters`.